lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -149,3 +149,8 @@ <h3 class="hd hd-3" id="header-${ section_data['section_key'] }">${ section_data
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform(iterationKey=".localized-datetime");
 </%static:require_module_async>
+
+<% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
+% if ga_acct:
```

Comment:
TLDR: remove conditional and `ga_acct` value checking...

While it's great that you're thinking about checking for whether a google analytics account is present before adding code that uses it, it's actually not desired here. The file you're including is super general, so an instance could conceivably want to include some HTML at the bottom of the instructor dashboard page that has nothing to do with GA...

Reply:
@caesar2164 I updated the file. Thanks!